documentation/userguide.md

```diff
@@ -0,0 +1,270 @@
+* Description of the application start and test procedure. This determines, whether the profiler must be registered in a start script or by setting user-specific environment variables. Furthermore, it must be determined if there are longer-running tests (e.g. overnight) that must be treated specially to get up-to-date results every day.
+
+## Development
+
+The application development lead must provide the following information: 
+
+* Definition whether the application has a client and/or server part and which of those should be included in the analysis. Depending on this, the profiler will need to be installed on the server and/or the client machines.
+* Which .NET framework version is used and whether the application is compiled as 32bit or 64bit assembly. This determines the profiler version to be used.
+* Incrementing version numbers in the build. The analysis can only reliably distinguish between different versions of the application if each version of the application has its own distinct version number. These numbers must be in ascending order.
+* Generating PDB files in the build. The analysis needs the PDB files to be able to map the coverage data from the binary files back to the source code. See the below section on builds for details.
+* Select exclusion criteria. Some assemblies or types might be irrelevant for the Test Gap analysis and therefore must be excluded. Examples are test code, which does not need to be tested itself, or automatically generated types whose code is deliberately used only partially, that do not change over time and were generated by a trustworthy program.
+
+## Operations
+
+Operations (DevOps) must perform the following tasks:
+
+* Installation of the profiler on the test machines (client and/or server, see the Installation section)
+* Facilitating access to the traces and binaries. These can either be made available on the test machine via a shared folder or transferred to a network share after test.
+* For client applications: installation of an Heartbeat application and activation of the profiler. Setup of a time-based execution. This enables a regular, automatic check of whether a client machine is able to send trace data to Teamscale.
+
+## Teamscale Administrator
+
+Once all the above-mentioned information is available and the Operations tasks have been completed, the Teamscale Administrator can create the dashboards:
+
+* Creation of one or more dashboards, depending on the test environment.
+* Customization of the dashboard, in order to collect the necessary data from the network share and start the preprocessing of the data.
+* For client applications: Creation of a Heartbeat dashboard and listing of all clients from which a regular Heartbeat trace is expected.
+* Setup of the exclusion criteria as parameters of the dashboard.
+
+
+# Client Heartbeat
```

Comment:
This is not part of the release, and we hardly ever use it anyway. Remove section?